youmubot-osu/src/discord/display.rs

```diff
@@ -274,64 +284,36 @@ mod scores {
                     })
                     .collect::<Vec<_>>();
 
-                let pw = pp.iter().map(|v| v.len()).max().unwrap_or(2);
-                /*mods width*/
-                let mw = plays.iter().map(|v| v.mods.str_len()).max().unwrap().max(4);
-                /*beatmap names*/
-                let bw = beatmaps.iter().map(|v| v.len()).max().unwrap().max(7);
-                /* ranks width */
-                let rw = ranks.iter().map(|v| v.len()).max().unwrap().max(5);
-
-                let mut m = serenity::utils::MessageBuilder::new();
-                // Table header
-                m.push_line(format!(
-                    " #  | {:pw$} | accuracy | {:rw$} | {:mw$} | {:bw$}",
-                    "pp",
-                    "ranks",
-                    "mods",
-                    "beatmap",
-                    rw = rw,
-                    pw = pw,
-                    mw = mw,
-                    bw = bw
-                ));
-                m.push_line(format!(
-                    "------{:-<pw$}--------------{:-<rw$}---{:-<mw$}---{:-<bw$}",
-                    "",
-                    "",
-                    "",
-                    "",
-                    rw = rw,
-                    pw = pw,
-                    mw = mw,
-                    bw = bw
-                ));
-                // Each row
-                for (id, (play, beatmap)) in plays.iter().zip(beatmaps.iter()).enumerate() {
-                    m.push_line(format!(
-                        "{:>3} | {:>pw$} | {:>8} | {:>rw$} | {} | {:bw$}",
-                        id + start + 1,
-                        pp[id],
-                        format!("{:.2}%", play.accuracy(self.mode)),
-                        ranks[id],
-                        play.mods.to_string_padded(mw),
-                        beatmap,
-                        rw = rw,
-                        pw = pw,
-                        bw = bw
-                    ));
-                }
-                // End
-                let table = m.build().replace("```", "\\`\\`\\`");
-                let mut m = serenity::utils::MessageBuilder::new();
-                m.push_codeblock(table, None).push_line(format!(
-                    "Page **{}/{}**",
-                    page + 1,
-                    self.total_pages()
-                ));
-                m.push_line("[?] means pp was predicted by oppai-rs.");
-                msg.edit(ctx, EditMessage::new().content(m.to_string()))
-                    .await?;
+                const SCORE_HEADERS: [&'static str; 6] =
+                    ["#", "pp", "accuracy", "ranks", "mods", "beatmap"];
+                const SCORE_ALIGNS: [Align; 6] = [Right, Right, Right, Right, Left, Left];
```

Comment:
Mods should be right-aligned (so `v2` suffixes line up)

Reply:
fixed